src/System.Data.SqlClient/ref/project.json

```diff
@@ -5,7 +5,8 @@
     "System.IO": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Threading.Tasks": "4.0.0",
-    "System.Xml.ReaderWriter": "4.0.0"
+    "System.Xml.ReaderWriter": "4.0.0",
+	"System.Collections.NonGeneric": "4.0.0",
```

Comment:
Is this new dependency on the legacy collections only due to the fact that `SqlBulkCopyColumnMappingCollection` derives from `System.Collections.CollectionBase`? Could `SqlBulkCopyColumnMappingCollection` in the contract be changed to just implement the `IList` interface so this contract doesn't have to depend on the legacy collections?

This would be consistent with other contracts which broke the dependency on `CollectionBase`, for example, [`X509CertificateCollection`](https://github.com/dotnet/corefx/blob/632d3e7db4400cab0c7076e6ebbc8413714adf4b/src/System.Security.Cryptography.X509Certificates/ref/System.Security.Cryptography.X509Certificates.Manual.cs#L19-L20) no longer derives from `CollectionBase`, only `IList` in the contract.